### PR TITLE
IslandTypes can now be added by addons

### PR DESCRIPTION
**Pull request by yrsegal**
This enables Iridescent floating flowers (https://github.com/L0neKitsune/Botanical-Addons/issues/67).

Has been tested.

### CODE REVIEW

File: src/main/java/vazkii/botania/common/item/ItemGrassSeeds.java [hunk 1]
```diff
@@ -166,13 +160,13 @@ public boolean onItemUse(ItemStack par1ItemStack, EntityPlayer par2EntityPlayer,
 
 	@SubscribeEvent
 	public void onTickEnd(TickEvent.WorldTickEvent event) {
-		if(event.phase == Phase.END) {
+		if(event.phase == TickEvent.Phase.END) {
 			int dim = event.world.provider.dimensionId;
 			if(blockSwappers.containsKey(dim)) {
 				List<BlockSwapper> swappers = blockSwappers.get(dim);
-
```
Comment (Vazkii) [inline src/main/java/vazkii/botania/common/item/ItemGrassSeeds.java]: why?

Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/ItemGrassSeeds.java]: Oh shit, intellij screwed up
I'll fix this D:

Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/ItemGrassSeeds.java]: Actually, these large blocks of tabs disappearing are cleanup from the last guy who edited this file. ¯\\\_(ツ)\_/¯

**Comment by yrsegal** [comment]
The last guy who pushed to ItemGrassSeeds (@blacksmithgu) left those tabs around, and my editor automatically cleaned them up.

**Comment by blacksmithgu** [comment]
It looks like your IDE or something else went through and changed a lot of whitespace lines. I can't see the diff which was commented on as you already changed it, but what were they changing from/to? Should I be leaving empty lines as empty with no tabs?

**Comment by blacksmithgu** [comment]
Looks like my project-specific cleanup script wasn't configured properly and wasn't being applied every time on save. That's fixed now; thanks for fixing it in the class I recently changed.

**Pull request merged**